game/src/ecs/components/HealthComponent.java

```diff
@@ -14,11 +14,16 @@
 import semanticAnalysis.types.DSLType;
 import semanticAnalysis.types.DSLTypeMember;
 
-/** The HealthComponent makes an entity vulnerable and killable */
+/**
+ * The HealthComponent adds health points and the ability to take damage and die to an entity. It
+ * stores the current health points and the maximal health points. For retrieving what killed or
```

Comment:
Gruppiere das mal zur besseren Lesbarkeit in Blöcke: einmal Health, einmal Damage und Cause of Damage, Call-Back für Death, Animationen.

Muss man noch mehr dazu wissen? Mit Blickwinkel eines PM-Studis: Wie die Sachen intern repräsentiert werden? Welche Systeme hier zugreifen? Passiert die Hit-Animation dann automatisch oder muss ich noch was tun? Dito die Sterbe-Sache?

Reply:
Ich habe es mal neu geschrieben und etwas strukturiert.

Reply:
👍